collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2888,6 +2886,34 @@ private void updateFieldListQuestions(FormIndex lastChangedIndex) {
         }
     }
 
+    private void saveAnswersForCurrentScreenOneByOneToSupportCalculations(FormEntryPrompt[] mutableQuestionsBeforeSave, List<ImmutableDisplayableQuestion> immutableQuestionsBeforeSave) {
```

Comment:
I think it's ok to overload `saveAnswersForCurrentScreen` instead of having a long name.

Reply:
Fixed.